README.md

```diff
@@ -9,11 +9,13 @@ Set Up
 To install the requirements first run the command
 
 ```
-sudo pip install -r requirements.txt
+sudo python3 setup.py install
 ```
-There is no `sudo` in Windows. Hence, you should run the command:
+
+then
+
 ```
-pip install -r requirements.txt
+sudo pip install -r requirements.txt
```

Comment:
I think this isn't required, bro. `setup.py` installs it already as the dependencies